bundle/manifests/ibm-auditlogging-operator.clusterserviceversion.yaml

```diff
@@ -506,12 +506,12 @@ spec:
     name: IBM
   relatedImages:
     - name: FLUENTD_IMAGE
-      image: quay.io/opencloudio/fluentd:v1.14.5-bedrock-1
+      image: quay.io/opencloudio/fluentd:v1.14.5-bedrock-2
     - name: AUDIT_GARBAGE_COLLECTOR_IMAGE
-      image: quay.io/opencloudio/audit-garbage-collector:1.8.0
+      image: quay.io/opencloudio/audit-garbage-collector:1.9.0
     - name: AUDIT_POLICY_CONTROLLER_IMAGE
-      image: quay.io/opencloudio/audit-policy-controller:3.13.0
+      image: quay.io/opencloudio/audit-policy-controller:3.14.0
     - name: IBM_AUDITLOGGING_OPERATOR_IMAGE
-      image: quay.io/opencloudio/ibm-auditlogging-operator:3.19.0
-  replaces: ibm-auditlogging-operator.v3.18.0
-  version: 3.19.0
+      image: quay.io/opencloudio/ibm-auditlogging-operator:3.20.0
+  replaces: ibm-auditlogging-operator.v3.19.0
```

Comment:
@PRTamilanban ,As per Adam's comment in slack, https://ibm-cloudplatform.slack.com/archives/G013X1K33AN/p1649773811518849, 
We will need to remove this replaces statement